wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <v-card>
+    <nav>
+      <v-layout>
+        <v-toolbar prominent>
+          <RouterLink to="/"><icon /></RouterLink>
+          <v-toolbar-title><RouterLink to="/">Wordle</RouterLink></v-toolbar-title>
+          <v-btn @click="showDialog = !showDialog"><settingsIcon /></v-btn>
```

Comment:
should shrink the width of the settings dialog

Reply:
I've reduced it to 500px.  I've made the same adjustment to all of the other dialog pages as well.  Thank you for pointing that out.